src/Resources.resx

```diff
@@ -210,4 +210,10 @@
   <data name="Cannot_specify_both_folder_and_workspace_options" xml:space="preserve">
     <value>Cannot specify both folder and workspace options.</value>
   </data>
+  <data name="Accepts_a_file_path_which_if_provided_will_produce_a_format_report_json_file_in_the_given_directory" xml:space="preserve">
+    <value>Accepts a file path, which if provided, will produce a `format-report.json` file in the given directory.</value>
```

Comment:
Since the filename won't always be `format-report.json` maybe change this to be `json report`